in_toto/slsa_provenance/v1.0/provenance.go

```diff
@@ -8,7 +8,7 @@ import (
 
 const (
 	// PredicateSLSAProvenance represents a build provenance for an artifact.
-	PredicateSLSAProvenance = "https://slsa.dev/provenance/v1?draft"
+	PredicateSLSAProvenance = "https://slsa.dev/provenance/v1"
```

Comment:
Since the predicate type will not include the minor version, can this package dir be just `v1` instead of `v1.0`?

> Minor version changes are always backwards compatible and “monotonic.” Such changes do not update the predicateType.

https://slsa.dev/provenance/v1#parsing-rules

Reply:
> My thinking is that if Provenance v1.1 introduces a change, it'd go in in_toto/slsa_provenance/v1/provenance.go. I'm wary of having in_toto/slsa_provenance/v1.0/provenance.go, in_toto/slsa_provenance/v1.1/provenance.go, etc. Since they're backwards compatible from here, I see no reason why we can't have a single v1/provenance.go.

SGTM and agreed!